packages/devextreme/js/__internal/grids/new/grid_core/data_controller/public_methods.test.ts

```diff
@@ -171,4 +167,65 @@ describe('PublicMethods', () => {
       expect(gridCore.totalCount()).toBe(4);
     });
   });
+
+  describe('getCombinedFilter', () => {
+    const innerSetup = ({ dataSourceFilter, columnFilterValues }) => setup({
+      dataSource: new DataSource({
+        store: {
+          type: 'array',
+          data: [{ a: 1 }, { a: 2 }, { a: 3 }],
+        },
+        filter: dataSourceFilter,
+      }),
+      columns: [
+        { dataField: 'a', filterValues: columnFilterValues },
+      ],
+    });
+
+    describe('when displayFilter and filter from dataSource are empty', () => {
+      it('should return empty filter', () => {
```

Comment:
Maybe just?:

```suggestion
   it('should return empty filter when displayFilter and filter from dataSource are empty', () => {
```

I doubt that there will be any other tests in this describe block.

Reply:
It's general rule of https://bettertests.js.org/